salt/modules/win_pkg.py

```diff
@@ -2310,15 +2298,7 @@ def _get_name_map(saltenv="base"):
     """
     Return a reverse map of full pkg names to the names recognized by winrepo.
     """
-    u_name_map = {}
-    name_map = get_repo_data(saltenv).get("name_map", {})
-
-    if not six.PY2:
-        return name_map
-
-    for k in name_map:
-        u_name_map[k] = name_map[k]
-    return u_name_map
+    return get_repo_data(saltenv).get("name_map", {})
```

Comment:
We still need to create a reverse map here. Should be:
```
    u_name_map = {}
    name_map = get_repo_data(saltenv).get("name_map", {})
    for k in name_map:
        u_name_map[k] = name_map[k]
    return u_name_map
```

Reply:
Good catch.  Even from my commit comment, I'm not 100% sure why I had to mess with this file four months ago (and then messed it up), but I believe it was to satisfy pre-commit requirements at the time. This file has no business being changed in this PR if it doesn't need to, so I'm going try to just rip the commit changing this out.

Interesting that no existing tests caught my mistake, this might be something to address.

Reply:
If it's meant to return the reverse map, I believe it was broken before...

```diff
-    u_name_map = {}
-    name_map = get_repo_data(saltenv).get("name_map", {})
-
-    if not six.PY2:
-        return name_map
-
-    for k in name_map:
-        u_name_map[k] = name_map[k]
-    return u_name_map
+    return get_repo_data(saltenv).get("name_map", {})
```

The reverse map would never be returned under py3.  So that's why this was taken out when I de-sixed it.

Reply:
@twangboy It looks like the original change is no longer required (the Windows tests pass), so I don't need to touch this module in this PR.  If the lack of the reverse map is a bug, that should be addressed in a separate PR.